src/index.ts

```diff
@@ -49,3 +54,25 @@ export function measureDependency (marker: IMarker, data = '', success = true):
   const telemetry = { dependencyTypeName, name, duration, success, data } as DependencyTelemetry
   trackDependency(telemetry)
 }
+
+interface RulesDictonary {
```

Comment:
@AmberWilkie This is cool. Can we add this all to a new file and import what's needed into index.ts?

Reply:
Ok, done.